src/parser.jl

```diff
@@ -1,3 +1,5 @@
+using Unicode
```

Comment:
I think you need to add this module to Project.toml by running `add Unicode`.

Reply:
Unicode was already a part of the project, so no change to make I think